src/blocks/product-card.css

```diff
@@ -0,0 +1,113 @@
+/* #region Product Card */
+.product-card {
+  display: flex;
+  flex-direction: column;
+
+  box-sizing: border-box;
```

Comment:
Set it to * selector

Reply:
Don't understand this. What do U mean by this?

Reply:
You've used `box-sizing` rule more than one time, so it better to set this property to * selector